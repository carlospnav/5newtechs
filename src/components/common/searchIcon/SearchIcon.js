import styled from "styled-components"

export default styled.img.attrs({
  src: "/static/search/icn-search.svg",
  alt: "Search Icon"
})`
  position: absolute;
  right: 20px;
  top: 16px;

  @media (min-width: 500px){
    right: 40px;
  }
`