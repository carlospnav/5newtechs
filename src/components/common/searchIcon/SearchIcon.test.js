import { shallow } from "enzyme"
import SearchIcon from "./SearchIcon"
import "jest-styled-components"

describe("<SearchIcon>", () => {
  it("should render a SearchIcon properly.", () => {
    const wrapper = shallow(<SearchIcon />)
    expect(wrapper).toMatchSnapshot()
  })
})