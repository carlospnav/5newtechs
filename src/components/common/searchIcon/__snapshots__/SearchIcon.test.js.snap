// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchIcon> should render a SearchIcon properly. 1`] = `
.c0 {
  position: absolute;
  right: 20px;
  top: 16px;
}

@media (min-width:500px) {
  .c0 {
    right: 40px;
  }
}

<img
  alt="Search Icon"
  className="c0"
  src="/static/search/icn-search.svg"
/>
`;
